cmake/beman.scope-config.cmake

```diff
@@ -0,0 +1,8 @@
+include(${CMAKE_CURRENT_LIST_DIR}/beman.scope-targets.cmake)
```

Comment:
do we need to open an issue against exemplar repo since an example of this doesn't appear?